components/pokemons/favourite-button.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <button @click.stop="handleFavourite" class="favourite-button">
+    <span v-if="isFavourite">Favourite</span>
+    <span v-else>unFavorite</span>
+  </button>
```

Comment:
```suggestion
  <button @click.stop="handleFavourite" class="favourite-button">
    {{ isFavourite ? 'Favourite' : 'unFavorite' }}
  </button>  
```
wouldn't this be better? less HTML node